server/integration_tests/test_data/demo_feb2023/query_6/chart_config.json

```diff
@@ -745,554 +745,6 @@
             ],
             "denom": "Count_Person",
             "title": "Mental Health Deaths by Age"
-          },
-          {
-            "columns": [
-              {
-                "tiles": [
-                  {
-                    "statVarKey": [
-                      "Count_Person_WithDisability_Female",
-                      "Count_Person_WithDisability_Male"
```

Comment:
why do we lose this?

Reply:
this one is interesting actually because it looks like these are stat vars from opening dc/topic/HealthEquity: https://screenshot.googleplex.com/9odTVhUvd6a5iUL

but dc/topic/HealthEquity has a score lower than 0.7 (the score threshold for uae model).

I think in this change, dc/topic/HealthEquity no longer gets opened because there is a new topic (dc/topic/WHOHealthBehavior) that comes before it

Reply:
Ah this is the sort of scenario I was worried about.  Though, in this case, the loss is ok it feels like?

Reply:
yeah I think the loss feels ok here! but are we even supposed to be opening topics that are lower than the score threshold?

Reply:
hmm, wait, no, we shouldn't be doing that... let me check.

Reply:
Could that be from rounding? https://screenshot.googleplex.com/7iJYjZ6StxfjAmK

Reply:
oh maybe .. I'll take a look

Reply:
Actually, could it be because HealthEquity is part of Health?  https://source.corp.google.com/piper///depot/google3/third_party/datacommons/schema/stat_vars/topics_health_insights.mcf;l=146?q=dc%2Ftopic%2FHealthEquity%20f:mcf

Reply:
oh, actually, that makes sense